### PR TITLE
Fix ktx issue

### PR DESCRIPTION
**Pull request by HifiExperiments**
fix https://github.com/vircadia/vircadia/issues/1137

Note: if you've already experienced the issue, you'll have to clear your ktx cache for things to get back to normal, I think.  but make sure to test by copying over a ktx_cache from the current release

### CODE REVIEW

File: libraries/gpu/src/gpu/Texture_ktx.cpp [hunk 1]
```diff
@@ -65,15 +65,14 @@ struct GPUKTXPayload {
     }
 
     bool unserialize(const Byte* data, size_t size) {
-        if (size != SIZE) {
```
Comment (HifiExperiments) [inline libraries/gpu/src/gpu/Texture_ktx.cpp:68]: the problem was that this size changed, so version 1 ktx files weren't getting parsed correctly

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-726222a-_d569eefb.dmg

**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-726222a-_d569eefb.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-726222a-.tar_d569eefb.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-726222a-.tar_d569eefb.Z

File: libraries/gpu/src/gpu/Texture_ktx.cpp [hunk 2]
```diff
@@ -88,8 +87,11 @@ struct GPUKTXPayload {
         memcpy(&_usageType, data, sizeof(TextureUsageType));
         data += sizeof(TextureUsageType);
 
-        memcpy(&_originalSize, data, sizeof(glm::ivec2));
-        data += sizeof(glm::ivec2);
+        if (version >= 2) {
+            memcpy(&_originalSize, data, sizeof(glm::ivec2));
+            data += sizeof(glm::ivec2);
+        }
```
Comment (ctrlaltdavid) [inline libraries/gpu/src/gpu/Texture_ktx.cpp:93]: Is the object re-used? If so, does `_originalSize` need to be set to `0, 0` for `version < 2`?

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-726222a-_d569eefb.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-726222a-.tar_d569eefb.gz
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-726222a-.tar_d569eefb.Z

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-726222a_d569eefb.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-Quest-Alpha-PR1141-726222a_d569eefb.apk

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-726222a-_d569eefb.exe

**Comment by ctrlaltdavid** [comment]
MacOS build failure is just a network timeout when building screenshare. There's a MacOS build (same commit) from the previous build attempt. So don't need to rebuild.

**Comment by SilverfishVR** [comment]
Testing Vircadia-PR1141-726222a
I have not seen the issue on this one, tested clean install, copy ktx cache from release, and reset cache.
👍 

I did loose the textures on woody at one point reloading stuff, but that has happened before so I don't think it's directly related.

**Comment by Aitolda** [comment]
Looks good to me.

File: libraries/gpu/src/gpu/Texture_ktx.cpp (hunk 2 above)
Comment (HifiExperiments) [reply libraries/gpu/src/gpu/Texture_ktx.cpp:93]: set default value above

**Comment by HifiExperiments** [comment]
no need to retest latest commit

**Review by ctrlaltdavid — APPROVED** [review]

**Comment by vircadia-build-notifier** [comment]
The following links are available: 
**build (ubuntu-18.04, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-0b97ab2-_ab1b69eb.sh
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-0b97ab2-.tar_ab1b69eb.Z
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-0b97ab2-.tar_ab1b69eb.gz

**build (macOS-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-0b97ab2-_ab1b69eb.dmg

**build (ubuntu-18.04, android)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-Quest-Alpha-PR1141-0b97ab2_ab1b69eb.apk
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-0b97ab2_ab1b69eb.apk

**build (windows-latest, full)**
 - https://athena-public.s3.eu-west-3.amazonaws.com/builds/PR/vircadia_vircadia/PR1141/Vircadia-PR1141-0b97ab2-_ab1b69eb.exe

**Pull request merged**